Exception trying to run benchmarks

I just checked out the source (master, 7e6e57a).  When I run `./gradlew benchmarks`, I get the following.  Looks like somehow it's bringing in a version of Guava that is too new to still have the `FutureFallback` class?

```
> Task :benchmarks FAILED
java.lang.NoClassDefFoundError: com/google/common/util/concurrent/FutureFallback
        at java.lang.Class.getDeclaredConstructors0(Native Method)
        at java.lang.Class.privateGetDeclaredConstructors(Class.java:2671)
        at java.lang.Class.getDeclaredConstructors(Class.java:2020)
        at com.google.inject.spi.InjectionPoint.forConstructorOf(InjectionPoint.java:243)
        at com.google.inject.internal.ConstructorBindingImpl.create(ConstructorBindingImpl.java:96)
        at com.google.inject.internal.InjectorImpl.createUninitializedBinding(InjectorImpl.java:629)
        at com.google.inject.internal.InjectorImpl.createJustInTimeBinding(InjectorImpl.java:845)

        at com.google.inject.internal.InjectorImpl.createJustInTimeBindingRecursive(InjectorImpl.java:772)
An unexpected exception has been thrown by the caliper runner.
        at com.google.inject.internal.InjectorImpl.createJustInTimeBindingRecursive(InjectorImpl.java:761)
Please see https://sites.google.com/site/caliperusers/issues
        at com.google.inject.internal.InjectorImpl.getJustInTimeBinding(InjectorImpl.java:256)
        at com.google.inject.internal.InjectorImpl.getBindingOrThrow(InjectorImpl.java:205)
        at com.google.inject.internal.InjectorImpl.getInternalFactory(InjectorImpl.java:853)
        at com.google.inject.internal.FactoryProxy.notify(FactoryProxy.java:46)
        at com.google.inject.internal.ProcessedBindingData.runCreationListeners(ProcessedBindingData.java:50)
        at com.google.inject.internal.InternalInjectorCreator.initializeStatically(InternalInjectorCreator.java:133)
        at com.google.inject.internal.InternalInjectorCreator.build(InternalInjectorCreator.java:106)
        at com.google.inject.internal.InjectorImpl.createChildInjector(InjectorImpl.java:217)
        at com.google.inject.internal.InjectorImpl.createChildInjector(InjectorImpl.java:224)
        at com.google.caliper.runner.CaliperMain.exitlessMain(CaliperMain.java:120)
        at com.google.caliper.runner.CaliperMain.main(CaliperMain.java:81)
        at com.google.caliper.runner.CaliperMain.main(CaliperMain.java:69)
        at com.google.re2j.Benchmarks.main(Benchmarks.java:206)
Caused by: java.lang.ClassNotFoundException: com.google.common.util.concurrent.FutureFallback
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:335)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 22 more

```

--- a/build.gradle
+++ b/build.gradle
@@ -43,7 +43,7 @@ dependencies {
   testCompile 'com.google.truth:truth:0.36'
 }
 
-configurations.testCompile {
+configurations.all {
   resolutionStrategy.eachDependency { DependencyResolveDetails details ->
     if (details.requested.group == 'com.google.guava' && details.requested.name == 'guava') {
       // Force guava 18.0, caliper:1.0-beta-2 requires it but truth:0.36 brings

